src/com/sahokia/kos/concerttickets/model/Client.java

```diff
@@ -0,0 +1,12 @@
+package com.sahokia.kos.concerttickets.model;
+
+public class Client extends User {
+
+    public void getTicket(Ticket ticket, String phone) {
+        ticket.share(phone);
+    }
+
+    public void getTicket(Ticket ticket, String phone, String email) {
+        ticket.share(phone, email);
```

Comment:
Does the ticket itself share something? 
Or the client? :)

Reply:
Changed the logic of share methods, moved them to Client class